### DISCUSSION TITLE
Add cancelRequest to UploadManager

### DISCUSSION THREAD

**Discussion by piersgill312** [livewire/livewire#3607] [3. Feature ideas]
Been Skimming through the 
`livewire/livewire/js/component/UploadManager.js` file and generally googling but there does not seem to be a way to cancel a current file upload request handled by livewire.

Is there a possibility to add such a function to the Uploadmanager?

I.e cancelRequest function etc.

I know there is a removeUpload but that is after the upload completes I believe?

**Comment by smares** [comment]
I would also like to know this, my application displays a modal for uploading files and I would like to cancel any ongoing uploads if the user decides to close the modal.

**Comment by joshhanley** [comment]
@piersgill312 and @smares can you have a look at and comment on PR #3216? Thanks!
